src/lib/containers/drug_upload_tool/Header.tsx

```diff
@@ -1,18 +1,26 @@
-import * as React from 'react';
+import * as React from 'react'
 
 export type HeaderProps = {
-  bodyText?: string; //should never take user input only use cardcoded text from a parent component. Uses dangerouslySetInnerHTML
-  title: string;
-};
+  isSubmitted?: boolean
+  bodyText?: string //should never take user input only use cardcoded text from a parent component. Uses dangerouslySetInnerHTML
+  title: string
+}
 
 export default function Header(props: HeaderProps) {
+  const bodyText = props.isSubmitted ? (
```

Comment:
IMO, the using the ternary operator twice inline makes this more challenging to parse.  If you agree, please rewrite to ease mental burden.

Reply:
Agree